src/precompiles/mod.rs

```diff
@@ -3,58 +3,82 @@ mod bn128;
 mod hash;
 mod identity;
 mod modexp;
+#[cfg(feature = "contract")]
 mod native;
 mod secp256k1;
+use evm::executor::PrecompileOutput;
+use evm::{Context, ExitError};
 
 use crate::precompiles::blake2::Blake2F;
 use crate::precompiles::bn128::{BN128Add, BN128Mul, BN128Pair};
 use crate::precompiles::hash::{RIPEMD160, SHA256};
 use crate::precompiles::identity::Identity;
 use crate::precompiles::modexp::ModExp;
+#[cfg(feature = "contract")]
 use crate::precompiles::native::{ExitToEthereum, ExitToNear};
 pub(crate) use crate::precompiles::secp256k1::ecrecover;
 use crate::precompiles::secp256k1::ECRecover;
-use crate::prelude::{Address, Vec};
-use evm::{Context, ExitError, ExitSucceed};
+use crate::prelude::Address;
+#[cfg(feature = "contract")]
+use crate::state::AuroraStackState;
+use crate::AuroraState;
 
 /// Exit to Ethereum precompile address (truncated to 8 bytes)
 ///
 /// Address: `0xb0bd02f6a392af548bdf1cfaee5dfa0eefcc8eab`
 /// This address is computed as: `&keccak("exitToEthereum")[12..]`
 const EXIT_TO_ETHEREUM_ID: u64 = 17176159495920586411;
 
+#[allow(dead_code)]
+pub fn exit_to_ethereum_address() -> Address {
+    Address::from_slice(
+        hex::decode("b0bd02f6a392af548bdf1cfaee5dfa0eefcc8eab")
```

Comment:
Just curious, where does this value come from?

Reply:
@birchmd take a look [there](https://github.com/aurora-is-near/aurora-engine/blob/060f31033880eaa7bc03b64e0237d7476db624db/src/precompiles/mod.rs#L23)

Reply:
Thanks @sept-en ! If we have a constant for it already, maybe the address could be derived using that instead of the hex string here?

Reply:
Yes, I think so. IMO I think even better is to have the address defined as the constant separately as well.

Reply:
I agree it shouldn't need to be computed at runtime if it is derived from a constant. Maybe a `const fn` could be used?

I think we should avoid having two separate constants defined independently though because otherwise in the future we may update one but forget to update the other.

Reply:
I like the idea of using the `const fn` approach!